client/src/landingPageContent/JoinUs.js

```diff
@@ -0,0 +1,129 @@
+import React, { useState, useEffect } from 'react';
+import { Container, Statistic, Transition, Segment, Step, Icon, Header, Image } from 'semantic-ui-react'
+import { makeCall } from '../apis';
+import {black, darkBlack, grey} from "../colors"
+
+export default function JoinUs (props) {
+    const [counts, setCounts] = useState({})
+    const [sendingRequest, setSendingRequest] = useState(false)
+    const fetchCounts = () => {
+        return makeCall({}, `/totalCounts`, 'get')
+            .then(countsResponse => {
+                if (countsResponse) {
+                    setCounts(countsResponse)
+                    setSendingRequest(false)
+                } else {
+                    // TODO: error
+                    // Add error toast when we introduce toasts to system
+                    setSendingRequest(false)
+                }
+            })
+    }
+
+    // Mounting
+    useEffect(() => {
+        setSendingRequest(true)
+        fetchCounts()
+    }, [props])
```

Comment:
Why does this need to rerun if the props change?

Reply:
Good point, I can vote to leave as is since this lets us polls the latest count every time the user comes to this tab, what do you think? I don't feel too strongly either way